### PR TITLE
PORTALS-2764, PORTALS-2763: fix all known vulnerabilities in pnpm audit (via overrides)

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
-settings:
-  autoInstallPeers: true
-  excludeLinksFromLockfile: false
-
 overrides:
+  word-wrap: ^1.2.4
+  semver: ^7.5.4
   '@types/react': 18.0.27
   goober: 2.1.9
   react-hot-toast: 2.2.0
```
Comment (jay-hodgson) [inline pnpm-lock.yaml:8]: Note the overrides already present (now represented in the root package)

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -1,10 +1,8 @@
 lockfileVersion: '6.0'
 
-settings:
```
Comment (jay-hodgson) [inline pnpm-lock.yaml:3]: This file was automatically generated/updated.  settings moved to the bottom of the file

File: package.json [hunk 3]
```diff
@@ -42,6 +42,13 @@
   "pnpm": {
     "patchedDependencies": {
       "@jsdevtools/ono@7.1.3": "patches/@jsdevtools__ono@7.1.3.patch"
+    },
+    "overrides": {
+      "word-wrap": "^1.2.4",
+      "semver": "^7.5.4",
+      "@types/react": "18.0.27",
+      "goober": "2.1.9",
+      "react-hot-toast": "2.2.0"
```
Comment (nickgros) [inline package.json:51]: Can you remove the `resolutions` field above which basically duplicates this?

Comment (jay-hodgson) [reply package.json:46]: Done

**Pull request merged**